state/runtime/evm/state.go

```diff
@@ -273,16 +273,16 @@ func (c *state) Len() int {
 }
 
 func (c *state) checkMemory(offset, size *big.Int) bool {
```

Comment:
I know this is not part of this task, but can we please change the name of this method to be something more descriptive, or add a comment explaining what it does (and what it returns)? 🙏